tests/test_files/KRYSOLVE.MAC

```diff
@@ -0,0 +1,109 @@
+!COM *****************************************************************************
+!COM KRYSOLVE.MAC                                                                *
+!COM                                                                             *
+!COM  This macro uses a KRYLOV subspace to solve a reduced harmonic              *
+!COM  analysis over a specified frequency range for a given number of            *
+!COM  frequency points (intervals).                                              *
+!COM                                                                             *
+!COM  COMMAND USAGE: KRYSOLVE,freqBeg,freqEnd,numFreq,outKey                     *
+!COM    freqBeg - starting value of the frequency range (Hz)                     *
+!COM    freqEnd - ending value of the frequency range (Hz)                       *
+!COM    numFreq - user specified number of intervals in frequency range          *
+!COM    loadKey - key specifying whether load should be ramped(0) or stepped(1)  *
+!COM    outKey  - [optional] key to output reduced solution to Yz.txt file       *
+!COM                                                                             *
+!COM *****************************************************************************
+
+!Store input arguments from user
+freqBeg=arg1
+freqEnd=arg2
+numFreq=arg3
+loadKey=arg4
+outKey=arg5
+
+!Check for illegal input values by user
+*IF,freqBeg,LT,0.0,THEN
+  *msg,error
+  The beginning frequency value for solving the reduced solution is required to be greater than or equal to 0
+  _nerr=1
+*ENDIF
```

Comment:
I like you use macros to check your code. Nice.

Watch out to execute this macro with values that will trigger ``*MSG``.